README.md

```diff
@@ -54,3 +54,12 @@ This imports the USWDS, pulls in the NJ-specific components and styles, and save
 This builds USWDS styles, builds the Fractal docs, and then deploys them to the gh-pages branch.
 
 Note: do not push directly to the gh-pages branch.
+
+## Releasing a new version to NPM
+
+1. Go to the Draft Release Github Action https://github.com/newjersey/njwds/actions/workflows/draft-release.yml
```

Comment:
Tiny nit: This readme has both "Github" and "GitHub" (capital "H").
It looks like the company prefers "GitHub". Please update that here, on line 64, and back on line 19 as well.